tests/tests_integration/test_api/test_data_workflows.py

```diff
@@ -618,20 +571,16 @@ def test_create_update_delete_data_modeling_trigger(
         assert updated_trigger.created_time == workflow_data_modeling_trigger.created_time
         assert updated_trigger.last_updated_time > workflow_data_modeling_trigger.last_updated_time
 
-    @pytest.mark.usefixtures("clean_created_sessions", "clean_created_workflow_triggers")
     def test_trigger_list(
         self,
         cognite_client: CogniteClient,
         workflow_scheduled_trigger: WorkflowTrigger,
         workflow_data_modeling_trigger: WorkflowTrigger,
     ) -> None:
-        triggers = cognite_client.workflows.triggers.get_triggers()
-        external_ids = {trigger.external_id for trigger in triggers}
+        listed = cognite_client.workflows.triggers.get_triggers(limit=-1)
```

Comment:
`triggers.get_triggers`

...why is this not `triggers.list` I wonder 🤔

Reply:
I wondered the same thing 🤪